frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -84,32 +84,34 @@ export function Experiment(): JSX.Element {
                         </Form.Item>
                         <Button onClick={handlePrev}>Go back</Button>
                     </div>
-                    <div className="confirmation">
-                        <PageHeader title={experiment?.name} />
-                        <div>{experiment?.description}</div>
-                        <div>Owner: {user?.first_name}</div>
-                        <div>Feature flag key: {experiment?.feature_flag}</div>
-                        <Row>
-                            <Col>
-                                <Row>Person allocation</Row>
-                                <Row>The following users will participate in the experiment</Row>
-                                <ul>
-                                    {experiment?.filters?.map((filter: PersonPropertyFilter, idx: number) => (
-                                        <li key={idx}>
-                                            Users with {filter.key} {filter.operator}{' '}
-                                            {Array.isArray(filter.value)
-                                                ? filter.value.map((val) => `${val}, `)
-                                                : filter.value}
-                                        </li>
-                                    ))}
-                                </ul>
-                            </Col>
-                        </Row>
-                        <Button onClick={createDraftExperiment}>Save as draft</Button>
-                        <Button type="primary" onClick={createExperiment}>
-                            Save and launch
-                        </Button>
-                    </div>
+                    {experiment && (
+                        <div className="confirmation">
+                            <PageHeader title={experiment.name} />
+                            <div>{experiment?.description}</div>
+                            <div>Owner: {user?.first_name}</div>
+                            <div>Feature flag key: {experiment?.feature_flag}</div>
+                            <Row>
+                                <Col>
+                                    <Row>Person allocation</Row>
+                                    <Row>The following users will participate in the experiment</Row>
+                                    <ul>
+                                        {experiment.filters?.map((filter: PersonPropertyFilter, idx: number) => (
```

Comment:
For the typing fix, this should probably instead be this?
```suggestion
                                        {experiment.filters?.properties.map((filter: PersonPropertyFilter, idx: number) => (
```